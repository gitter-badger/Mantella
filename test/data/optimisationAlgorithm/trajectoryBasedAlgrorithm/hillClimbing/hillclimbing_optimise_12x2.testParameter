	0.0		-5.1	7.0		 6.85	-2.89	6.89	 0.586	 5.0	 5.0	 5.0	 4.89	1.0
	0.0		-5.1	7.0		-3.58	 4.97	2.45	-1.48	-0.48	-4.25	-5.0	-5.0	1.0