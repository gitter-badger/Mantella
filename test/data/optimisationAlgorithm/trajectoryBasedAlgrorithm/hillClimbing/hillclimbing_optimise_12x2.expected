ARMA_MAT_TXT_FN008
2 12
   0.000000000000e+00  -3.200000000000e+00   5.000000000000e+00   5.000000000000e+00  -2.890000000000e+00   5.000000000000e+00   5.860000000000e-01   5.000000000000e+00   5.000000000000e+00   5.000000000000e+00   4.890000000000e+00   1.000000000000e+00
   0.000000000000e+00  -5.000000000000e+00   5.000000000000e+00  -3.580000000000e+00   4.970000000000e+00   2.450000000000e+00  -1.480000000000e+00  -4.800000000000e-01  -4.250000000000e+00  -5.000000000000e+00  -5.000000000000e+00   1.000000000000e+00
